cron_tasks_test.go

```diff
@@ -0,0 +1,88 @@
+package scalingo
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	gomock "github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCronTasksClient(t *testing.T) {
```

Comment:
You are just testing the `CronTasksAdd` method in this function.
```suggestion
func TestCronTasksClient_CronTasksAdd(t *testing.T) {
```